frontend/editor.css

```diff
@@ -913,11 +913,17 @@ pluto-trafficlight {
 pluto-trafficlight::after {
     content: "";
     /* Calc needs units everywhere to work */
-    top: calc(0px - 2 * var(--patternHeight));
+    top: calc(0px - 10 * var(--patternHeight));
     left: 0;
     width: 100%;
-    height: calc(100% + 2 * var(--patternHeight));
+    height: calc(100% + 10 * var(--patternHeight));
     position: absolute;
+    opacity: 0;
+}
+
+/* This class will toggle animation. Other classes will make animation visible with opacity */
+pluto-cell.activate_animation pluto-trafficlight::after {
+    animation: 10s linear 100ms infinite running scrollbackground;
```

Comment:
100ms?

Reply:
Will switch to 0. That made some sense when animation-triggering state updates were not batched behind `activate_animation`.